docs/ceci_dragon2.md

```diff
@@ -0,0 +1,20 @@
+# nf-core/configs: Dragon2 CÉCI Cluster Configuration
+
+This configuration provides sensible default parameters to run nf-core pipelines (and nf-core compatible pipelines) on the [CÉCI Dragon2 cluster](https://www.ceci-hpc.be/clusters.html#dragon2).
+To use it, add `-profile ceci_dragon2` when running a pipeline.
+This will automatically download the [`ceci_dragon2.config`](../conf/ceci_dragon2.config).
+
+The configuraiton sets `slurm` as the default executor and enables `singularity` as container runner.
+
+## Loading the required modules
+
+Before running a pipeline, one should load `nextflow` using the environment module system.
+This can be achieved with:
+
+```bash
+# Load modules
+module load 'Nextflow/24.04.2'
```

Comment:
Does singularity also need to be laoded, or is this available by default?

Also the dragon2 config doesn't specify a singualrity cache directory, if users need to do this manually maybe document to specify this with the nextflow `NXF_SINGULARITY_CACHEDIR` (https://nextflow.io/docs/latest/reference/env-vars.html) so they at least can cache on a per-user basis

Reply:
Yes, the `singularity` command is available by default on both CÉCI clusters (Dragon2 and NIC5).

Since there is no shared directory that I am aware of on those clusters, is this recommended to still specify it to something like `${HOME}/.cache/singularity`? Or is it okay to leave it empty so that the system uses its default value?

Reply:
It is recommended to specify a directory with `NXF_SINGULARITY_CACHE_DIR` as the 'default' by Nextflow is to pull the images into the `work/` directory, which is run specific. On the next run where `-w` is somewhere else it'll download the images again

Reply:
Thanks for the information, I added a default cache directory (for consistency across all users). If, in the future, a shared directory is created on these clusters, I'll document how to use it/switch to it.